eth1data/eth1data.go

```diff
@@ -228,9 +234,18 @@ func GetEth1Transaction(hash common.Hash) (*types.Eth1TxData, error) {
 	}
 
 	if txPageData.BlockNumber != 0 {
-		err := db.ReaderDb.Get(&txPageData.Epoch,
-			`select epochs.finalized, epochs.globalparticipationrate from blocks left join epochs on blocks.epoch = epochs.epoch where blocks.exec_block_number = $1 and blocks.status='1';`,
-			&txPageData.BlockNumber)
+		latestFinalizedEpoch := services.LatestFinalizedEpoch()
+		err := db.ReaderDb.Get(&txPageData.Epoch, `
```

Comment:
This looks similar to SELECT a few lines above, but is different formatted. Why?

Reply:
Ah yes, moved the query to the db.go and and call it now from both places.